docs/01_nodeos/03_plugins/http_client_plugin/index.md

```diff
@@ -7,31 +7,10 @@ The `http_client_plugin`  is an internal utility plugin, providing the `producer
 ```console
 # config.ini
 plugin = eosio::http_client_plugin
-https-client-root-cert = "path/to/my/certificate.pem"
-https-client-validate-peers = 1
 ```
 ```sh
 # command-line
-nodeos ... --plugin eosio::http_client_plugin  \
-           --https-client-root-cert "path/to/my/certificate.pem"  \
-           --https-client-validate-peers 1
-```
-
-## Options
-
-These can be specified from both the `nodeos` command-line or the `config.ini` file:
-
-```console
-Config Options for eosio::http_client_plugin:
-  --https-client-root-cert arg          PEM encoded trusted root certificate 
-                                        (or path to file containing one) used 
-                                        to validate any TLS connections made.  
-                                        (may specify multiple times)
-                                        
-  --https-client-validate-peers arg (=1)
-                                        true: validate that the peer 
-                                        certificates are valid and trusted, 
-                                        false: ignore cert errors
+nodeos ... --plugin eosio::http_client_plugin 
```

Comment:
I wonder if we can just remove this whole doc page. With the removal of its configs it's pretty much just an implementation detail at this point. idk if maybe @lparisc or @iamveritas has opinion on it

Reply:
Indeed, with https removed, there's no need for an `http_client_plugin` explainer doc. @greg7mdp feel free to remove this file and any reference to it (to reduce PR overhead), or we can remove in another PR if you prefer. Thanks for letting us know @spoonincode

Reply:
Thanks @spoonincode and @lparisc , I have removed the page.

Reply:
You need to remove the reference to the page in:
`docs/01_nodeos/03_plugins/index.md`